.changesets/remove-moped-support.md

```diff
@@ -0,0 +1,7 @@
+---
+bump: "patch"
```

Comment:
If we're dropping support for something, maybe it should not be a `patch` release? @tom thoughts?

Reply:
It should be at least a minor release like Jeff says. I wouldn't want to make this a major version release.

Reply:
https://docs.appsignal.com/ruby/integrations/mongodb.html#older-mongoid-versions-and-mongomapper I think we virtually removed Moped support in the past. This PR removes the last trace of it left; that's why I chose to go with a patch.